projects/igniteui-angular/src/lib/grids/grid/grid-row-editing.spec.ts

```diff
@@ -1881,21 +1806,20 @@ describe('IgxGrid - Row Editing #grid', () => {
             expect(columns[2].editable).toBeTruthy(); // column.editable set to true
             expect(columns[3].editable).toBeTruthy(); // column.editable not set
             expect(columns[4].editable).toBeFalsy();  // column.editable set to false
-        }));
+        });
 
-        it(`Default column editable value is correct, when row editing is disabled`, fakeAsync(() => {
-            const fix = TestBed.createComponent(IgxGridTestComponent);
+        it(`Default column editable value is correct, when row editing is disabled`, () => {
+            fix = TestBed.createComponent(IgxGridTestComponent);
```

Comment:
This test can be made with the IgxGridWithEditingAndFeaturesComponent and should be moved to navigation describe